packages/devtools_app/test/scaffold_test.dart

```diff
@@ -158,22 +159,25 @@ void main() {
 
       const debuggerScreenKey = Key('debugger screen');
       const debuggerTabKey = Key('debugger tab');
-      await tester.pumpWidget(wrapWithControllers(
-        DevToolsScaffold(
-          tabs: const [
-            _TestScreen(
-              DebuggerScreen.id,
-              debuggerScreenKey,
-              tabKey: debuggerTabKey,
-              showFloatingDebuggerControls: false,
+      await tester.pumpWidget(
+        wrapWithAnalyticsProvider(
+          wrapWithControllers(
```

Comment:
can you make wrapWithControllers support the analytics providers to reduce the boilerplate in writing these tests?

Reply:
done